### ISSUE TITLE
[suggestion] option to retrieve books from librarian trade pool?

### ISSUE THREAD

**Issue by yumekazu** [AlexModGuy/DomesticationInnovation#94]
Hi o/
I'm building up my own modpack in 1.18.2 version, and I wanted to add your mod as I think more usefull pets could be a great addition but I've some problem with it.

Even though you have your own PNJ (animal tamer), you put your enchanted books in the Librarian trades pool and it's getting in the way to have traditional enchantments (as you add many more enchantments, nearly half of the librarian trades are yours and it's getting on the way to progression).
Plus, in order to reduce the high number of your books in the Librarian, I've turned off a good half of your books in the config options, but they are still showing in Librarian pool.

So my suggestion is: would it be possible to have an option to put all your books in your PNJ and not in the Librarian ?
Or an option to switch off the Librarian trades you add?

That would be awesome 🙂

**Comment by cassowarycasper** [comment]
Yeah, seconding this. it would be nice if the book trades were able to be restricted to the new Animal Tamer villagers. Not being able to toggle off librarians having them is a little frustrating. The abilities are all really fun though, and I am having a blast!!

**Comment by EarthKone** [comment]
Having this issue too, getting really annoying
